### ISSUE TITLE
weighting 'from_simprod' function tries to load missing values (Trac #1356)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1348]
icecube.weighting.weighting.from_simprod (weighting.py L578)
after connecting to the simprod database, the function tries to load 'NUGEN::elogmin', 'NUGEN::elogmax', and 'NUGEN::injectionradius', which do not exist in new neutrino generator tables.

Migrated from https://code.icecube.wisc.edu/ticket/1356
```json
{
    "status": "closed", 
    "changetime": "2015-09-18T18:29:44", 
    "description": "icecube.weighting.weighting.from_simprod (weighting.py L578)\nafter connecting to the simprod database, the function tries to load 'NUGEN::elogmin', 'NUGEN::elogmax', and 'NUGEN::injectionradius', which do not exist in new neutrino generator tables.", 
    "reporter": "ddouglas", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1442600984726412", 
    "component": "combo reconstruction", 
    "summary": "weighting 'from_simprod' function tries to load missing values", 
    "priority": "normal", 
    "keywords": "weighting, simprod", 
    "time": "2015-09-18T18:15:17", 
    "milestone": "", 
    "owner": "jvansanten", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150918T18:20:08: ddouglas commented:

> diff from my quick fix:
> 
> ```text
> 568a569,577
> > 		try:
> > 			EnergyMinLog = steering['NUGEN::elogmin']
> > 			EnergyMaxLog = steering['NUGEN::elogmax']
> > 			InjectionRadius = steering['NUGEN::injectionradius']*I3Units.m
> > 		except KeyError:
> > 			EnergyMinLog = numpy.log10(steering['NUGEN::from_energy'])
> > 			EnergyMaxLog = numpy.log10(steering['NUGEN::to_energy'])
> > 			InjectionRadius = steering['NUGEN::cylinder_radius']*I3Units.m
> > 
> 570,571c579,580
> < 		    EnergyMinLog   =steering['NUGEN::elogmin'],
> < 		    EnergyMaxLog   =steering['NUGEN::elogmax'],
> ---
> > 		    EnergyMinLog   =EnergyMinLog,
> > 		    EnergyMaxLog   =EnergyMaxLog,
> 574c583
> < 		    InjectionRadius=steering['NUGEN::injectionradius']*I3Units.m,
> ---
> > 		    InjectionRadius=InjectionRadius,
> ```
>

**Comment by icecube-trac** [comment]
Trac update at 20150918T18:29:44: 
* jvansanten changed status from "new" to "closed"
* jvansanten changed resolution from "" to "fixed"
* jvansanten changed owner from "" to "jvansanten"
* jvansanten commented:

> In changeset 137748:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="137748"
> Chase churn in NuGen iceprod module. Fixes #1348.
> ```
